modules/@apostrophecms/ui/ui/apos/components/AposTagApply.vue

```diff
@@ -154,32 +159,9 @@ const isTagFound = computed(() => {
 });
 
 const noTagsTranslation = computed(() => {
-  return props.tags.length
-    ? $t('apostrophe:tagNoResultFor', { tag: searchValue.value.data })
-    : $t('apostrophe:tagNoTagsYet');
-});
-
-// Sort checked first
-// then indeterminate
-// and finally unchecked alphabetically
-const sortedTags = computed(() => {
-  if (!applyToIds.value.length) {
-    return props.tags;
-  }
-
-  const checked = props.tags.filter(tag =>
-    checkboxes.value[tag.slug].model.value === true &&
-    checkboxes.value[tag.slug].choice.indeterminate !== true
-  );
-  const indeterminate = props.tags.filter(tag =>
-    checkboxes.value[tag.slug].model.value === true &&
-    checkboxes.value[tag.slug].choice.indeterminate === true
-  );
-  const unchecked = props.tags.filter(tag =>
-    checkboxes.value[tag.slug].model.value !== true
-  );
-
-  return [].concat(checked, indeterminate, unchecked);
+  return !props.tags.length && !searchValue.value.data
+    ? $t('apostrophe:tagNoTagsYet')
+    : $t('apostrophe:tagNoResultFor', { tag: searchValue.value.data });
```

Comment:
Makes `No results found` having precedence over `No tags yet` when the user typed something.